triggermail_templates.py

```diff
@@ -128,6 +134,53 @@ def dissect_filename(self, template_filename):
         self.partner = self.settings.get("partner", self.partner) or self.partner
         self.partner = self.partner.replace("_templates", "")
 
+    def generate_pertinent_file_names_and_map(self, template_filename):
+        template_filename = template_filename.replace(self.path + '/', '')
+        image_regex = "/img/" + self.partner + "/" + "(.*?)['\"]"
```

Comment:
Combine the `"/"` with the rest into `"/img/" + self.partner + "/(.*?)['\"]"` ? slightly simpler

also to be anal-retentive: self.partner might include characters that are treated specially in regexes; I know it isn't *supposed* to, but if it does will this break in some catastrophic way? It might be fine.

You could use `re.compile` to check the pattern for errors here, and to compile it once, but for something like this: do whatever you think is simpler.